cardano-cli/src/Cardano/CLI/EraBased/Options/Transaction.hs

```diff
@@ -235,8 +235,8 @@ pTransactionBuildEstimateCmd era _envCli = do
                       "Filepath of auxiliary script(s)")
           <*> many pMetadataFile
           <*> pFeatured (shelleyBasedToCardanoEra sbe) (optional pUpdateProposalFile)
-          <*> pVoteFiles sbe AutoBalance
-          <*> pProposalFiles sbe AutoBalance
+          <*> pVoteFiles sbe ManualBalance
```

Comment:
Should we add an information in the help text that the balancing method has changed?